lib/webpacker/compiler.rb

```diff
@@ -13,6 +13,7 @@ def initialize(webpacker)
 
   def compile
     if stale?
+      Webpacker.clobber
```

Comment:
Seems like perhaps "clobber" should be a method on the compiler which is then called from Webpacker. This kinda of reaching is odd.